runtime/compiler/il/symbol/J9ResolvedMethodSymbol.hpp

```diff
@@ -43,6 +43,9 @@ namespace J9
 
 class OMR_EXTENSIBLE ResolvedMethodSymbol : public OMR::ResolvedMethodSymbolConnector
    {
+public:
+   TR::KnownObjectTable::Index getKnownObjectIndexForParm(int32_t slot);
```

Comment:
doxygen? and what does slot mean - since you say parm are we counting the receiver or not - I am assuming yes from the implementation. Is there a generalization of this for any method handle being passed through function to get to an invoke of it? Judging from the above I suspect you can fix it at the invoke site, but I wanted to check because if not there is an obvious extension that should be considered as an immediate follow-on

Reply:
Changed `slot` to `ordinal` to avoid confusion. 
There is indeed a follow-on item dealing with the known objects being passed as arguments from invoke site in inliner.